zh-CN/firefox_privacy_notice.md

```diff
@@ -26,7 +26,7 @@
 
 * __位置数据__：首次使用 Firefox 时，Firefox 会使用 IP 地址根据您所在的国家/地区设置默认搜索提供商。[了解更多信息](https://support.mozilla.org/kb/change-your-default-search-settings-firefox)。
 
-* __搜索查询__：默认情况下，如果您选择的搜索提供商支持搜索建议，Firefox 会将搜索查询发送给您的搜索提供商，以帮助您发现其他用户搜索的常用短语并改善您的搜索体验。[了解详情](https://support.mozilla.org/kb/search-suggestions-firefox)，包括如何禁用此功能。如果启用 Firefox Suggest，我们和我们的合作伙伴也可能会收到您的搜索查询。[在下方了解更多信息](#searches)。
+* __搜索查询__：默认情况下，如果您选择的搜索提供商支持搜索建议，Firefox 会将搜索查询发送给您的搜索提供商，以帮助您发现其他用户搜索的常用短语并改善您的搜索体验。[了解详情](https://support.mozilla.org/kb/search-suggestions-firefox)，包括如何禁用此功能。我们的默认搜索提供商的链接如下：

```

Comment:
This translation is incomplete.

Where are the links and the sentence after the link??

Reply:
Fixed